eliHenderson/assignments/fastSlow/lc141.js

```diff
@@ -0,0 +1,69 @@
+// Problem Statement #
+
+// Given the head of a Singly LinkedList, write a function to determine if the LinkedList has a cycle in it or not.
+/**
+ * process: two pointer
+ * one faster pointer and one slow pointer
+ * 1 => 2=> 3=> 4=>5=>6=>3
+ *
+ * set both pointer as the head node
+ * increment them from the head
+ *
+ * while loop
+ *  keep incrementing until either
+ *  one node reaches null or
+ *  the two nodes meet
+ *
+ * time complexity: O(N)
+ * space complexity: O(1)
```

Comment:
Nice high-level approach, and even adding the Big O notation for efficiency. What edge cases could there possibly be?